### DISCUSSION TITLE
How to get a projected view of a mesh

### DISCUSSION THREAD

**Discussion by T4mmi** [pyvista/pyvista#2756] [Q&A] [answered]
Hi, I'm trying to get a "simplified projection" of a mesh (any mesh actually).  
I've seen the `project_points_to_plane` helper but it "just" reproject points, it doesn't mask non visible faces...

What i want is basically a `vtkHardwareSelector` that would refine cells partially visible ... the idea is to be able to provide a simplified mesh from a complex multiblock with a point of view (i.e. top view, side views ...).  I did tried something based on ray tracing but my implementation was horribly slow and didn't even refine faces .

**Comment by tkoyama010** [comment, marked as answer]
If you have a vtk object that you would like PyVista to support, please make a suggestion via [feature request](https://github.com/pyvista/pyvista/issues/new?assignees=&labels=feature-request&template=feature-request.yml).

**Reply by ShawnjpWang** [reply]
Yeay.I have the same question and supply in the  feature request module. Maybe you can add a comment on the issue. Or have you found another way to fulfill your thoughts?
